requirements/dev.txt

```diff
@@ -21,7 +21,7 @@ elasticsearch-dsl==6.1.0
 elasticsearch==6.3.0      # via elasticsearch-dsl
 enum34==1.1.6             # via astroid
 first==2.0.1              # via pip-tools
-futures==3.2.0            # via caniusepython3, isort
+futures==3.1.0            # via caniusepython3, isort
```

Comment:
this downgrade was actually necessary for python 3.7 local test development. it's not a runtime dependency, just used for locally bringing up the test project.